content/docs/setup/kubernetes/download/index.md

```diff
@@ -51,3 +51,51 @@ To use the Istio release Helm chart repository, add the Istio release repository
 {{< text bash >}}
 $ helm repo add istio.io https://storage.googleapis.com/istio-release/releases/{{< istio_full_version >}}/charts/
 {{< /text >}}
+
+## `istioctl` auto-completion
+
+If you are using Bash or Zsh, `istioctl` includes a tab completion file that you can use. 
+
+### Option 1. Installing `istioctl` on Linux
+
+Both the Bash and Zsh completion files are located in the `tools` directory. Copy the preferred completion file to your home directory, and add the following line to source the `istioctl` completion file in your `.bashrc` file. For example with Bash, you will be adding the following line:
+
+{{< text bash >}}
+$ source ~/istioctl.bash
+{{< /text >}}
+
+### Option 2. Installing `istioctl` on OSX
+
+If you are using OSX with the Bash shell, make sure that the bash-completion package is installed. If you are using [Homebrew](https://brew.sh), you can check to see if the bash-completion package is installed with the following command:
+
+{{< text bash >}}
+$ brew info bash-completion
+{{< /text >}}
+
+If installed, the output from this command should display output similar to the following:
+
+{{< text bash >}}
+$ brew info bash-completion
+bash-completion: stable 1.3 (bottled)
+{{< /text >}}
+
+If it is _not_ installed, proceed with installing the bash-completion package with the following command:
+
+{{< text bash >}}
+$ brew install bash-completion
+{{< /text >}}
+
+{{< tip >}}
+Once the auto-completion package has been installed on your OSX system, also make sure you add the following line of code to your `~/.bash_profile` file:
+
+`[ -f /usr/local/etc/bash_completion ] && . /usr/local/etc/bash_completion`
+{{< /tip >}}
+
+## Using `istioctl` auto-completion
+
+If the `istioctl` completion file has been installed correctly, press the Tab key while writing an `istioctl` command, and it should return a set of command suggestions for you to choose from:
```

Comment:
It doesn't work for me ... how do I check if the "`istioctl` completion file has been installed correctly"?

Reply:
@frankbu if everything else is in place, and the `istioctl.bash` file has been sourced, it should just...work. However, we should probably come up with copy that adds a bit of troubleshooting in case this happens....